tests/failpoints/cases/test_unsafe_recovery.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::iter::FromIterator;
+use std::sync::{Arc, Condvar, Mutex};
+
+use futures::executor::block_on;
+use pd_client::PdClient;
+use raftstore::store::util::find_peer;
+use test_raftstore::*;
+
+#[test]
+fn test_unsafe_recover_send_report() {
```

Comment:
Please add some comments in the test, to describe the purpose of every paragraph of the test code.

Reply:
Done